projects/storefrontapp-e2e-cypress/cypress/helpers/variants/apparel-checkout-flow.ts

```diff
@@ -19,7 +19,9 @@ export function addVariantOfSameProductToCart() {
   cy.server();
   cy.route(
     'GET',
-    `/rest/v2/apparel-uk-spa/products/${products[1].code}/reviews*`
+    `${Cypress.env('OCC_PREFIX')}/apparel-uk-spa/products/${
```

Comment:
`apparel-uk-spa` - Shouldn't we use `BASESITE` env variable mutated previously to `APPAREL_BASESITE`?

Reply:
Thanks, i'll change it! I was thinking it was fine as it was an apparel test. 

Had to mutate the `base_site` because of the functions being used in checkout-flow and another one that I forgot